activation/activation.go

```diff
@@ -559,3 +560,29 @@ func (b *Builder) GetPositioningAtx(epochId types.EpochId) (*types.ActivationTxH
 	}
 	return posAtx, nil
 }
```

Comment:
@noam moved here from types to avoid types being dependent of signing (circular dependency)